frontend/tests/components/opportunity/OpportunityDescription.test.tsx

```diff
@@ -108,17 +112,19 @@ describe("OpportunityDescription", () => {
     const mailtoLink = screen.getByRole("link", {
       name: "Contact Email Description",
     });
-    expect(mailtoLink).toHaveAttribute(
-      "href",
-      `mailto:${mockOpportunityData.summary.agency_email_address}`,
-    );
+    const agency_email = mockOpportunityData.summary.agency_email_address
+      ? mockOpportunityData.summary.agency_email_address
+      : "";
+    expect(mailtoLink).toHaveAttribute("href", `mailto:${agency_email}`);
 
     const telLink = screen.getByRole("link", {
-      name: mockOpportunityData.summary.agency_phone_number,
+      name: mockOpportunityData.summary.agency_phone_number
+        ? mockOpportunityData.summary.agency_phone_number
+        : "",
     });
-    expect(telLink).toHaveAttribute(
-      "href",
-      `tel:${mockOpportunityData.summary.agency_phone_number.replace(/-/g, "")}`,
-    );
+    const number = mockOpportunityData.summary.agency_phone_number
+      ? mockOpportunityData.summary.agency_phone_number.replace(/-/g, "")
+      : "";
+    expect(telLink).toHaveAttribute("href", `tel:${number}`);
```

Comment:
do we want to add a test supplying null data to make sure the defaulting / null checking behavior is working right?

Reply:
Good question. I'd like to punt this to https://github.com/HHS/simpler-grants-gov/issues/2321 as I'm not sure the correct pattern.